templates/vue/src/components/modals/NodeModal/forms/ContentForm/ActivityForm/index.vue

```diff
@@ -390,6 +409,15 @@ export default {
       question.followUp.questionId = ""
       question.followUp.nodeId = ""
     },
+    setCustomConfirmation(question, enabled) {
+      if (enabled) {
+        question.confirmation.title = ""
+        question.confirmation.message = "<p></p>"
+      } else {
+        question.confirmation.title = ""
+        question.confirmation.message = ""
+      }
+    },
```

Comment:
It could be helpful to add a comment explaining why the custom confirmation toggle is implemented this particular way.